src/table/components/head/TableHeadWrapper.tsx

```diff
@@ -52,58 +36,37 @@ function TableHeadWrapper({ tableData, changeSortOrder, areBasicFeaturesEnabled
         {columns.map((column, columnIndex) => {
           // The first cell in the head is focusable in sequential keyboard navigation,
           // when nebula does not handle keyboard navigation
-          const tabIndex = columnIndex === 0 && !keyboard.enabled ? 0 : -1;
           const isCurrentColumnActive = layout.qHyperCube.qEffectiveInterColumnSortOrder[0] === column.colIdx;
           const ariaSort = isCurrentColumnActive ? FullSortDirection[column.sortDirection] : undefined;
 
-          const handleKeyDown = (evt: React.KeyboardEvent) => {
-            handleHeadKeyDown({
-              evt,
-              rootElement,
-              cellCoord: [0, columnIndex],
-              column,
-              changeSortOrder,
-              isInteractionEnabled,
-              setFocusedCellCoord,
-              areBasicFeaturesEnabled,
-            });
-          };
+          // const handleKeyDown = (evt: React.KeyboardEvent) => {
+          //   handleHeadKeyDown({
+          //     evt,
+          //     rootElement,
+          //     cellCoord: [0, columnIndex],
+          //     column,
+          //     changeSortOrder,
+          //     isInteractionEnabled,
+          //     setFocusedCellCoord,
+          //     areBasicFeaturesEnabled,
+          //   });
+          // };
 
           return (
             <StyledHeadCell
               headerStyle={headerStyle}
               key={column.id}
               align={column.align}
               className="sn-table-head-cell sn-table-cell"
-              tabIndex={tabIndex}
               aria-sort={ariaSort}
-              aria-pressed={isCurrentColumnActive}
```

Comment:
I'm pretty sure we don't need this. It is mostly used for checkboxes and stuiff. aria-sort is undefined if it is not sorted on that column, should be enough